std/algorithm/iteration.d

```diff
@@ -4289,3 +4304,625 @@ unittest
          [1, 2, 0],
          [2, 1, 0]]));
 }
+
+/**
+Computes the $(WEB https://en.wikipedia.org/wiki/Binomial_coefficient, binomial coefficient)
+of $(D n,k).
+It is also known as "n choose k " or more formally as $(D n!/k!(n-k)).
+
+Params:
+    n = non negative integer
+    k = non negative integer, k <= n
```

Comment:
While I think this is fine too, pre- and post-conditions are often put in their own sections, ala
```
Preconditions: `k <= n`
```
Note that since `n` and `k` are parameter names, all instances of these as standalone words in the documentation comment will be expanded as `$(D_PARAM …)`, which for dlang.org makes them italic. You can circumvent this by prepending the word with an underscore, like `_n`. You can compare the generated docs for what makes most sense.